src/17-infinite-discovery/data.md

```diff
@@ -1,5 +1,48 @@
 # How to generate the data
 
+## Quick way to generate the data
+
+## Load the production console
+
+```bash
+  bundle exec rails console production
+```
+
+```ruby
+  # assuming you want to generate the data for the artworks in the curators picks collection
+  collection = MarketingCollection.find('curators-picks')
+  artworks = collection.artworks.published.for_sale
+  artworks_json = JSON.pretty_generate(artworks.map do |w|
+    {
+      id: w.id,
+      slug: w.slug,
+      title: w.title,
+      date: w.date,
+      rarity: w.attribution_class,
+      medium: w.category,
+      materials: w.medium,
+      list_price_amount: w.price_listed,
+      list_price_currency: w.price_currency,
+      categories: w.total_genome.without("Art", "Career Stage Gene").select{ |k,v| k !~ /(galleries based|made in)/i && v == 100}.keys,
```

Comment:
I don't know our genome's well, so just curious what the filtering is doing here?

Reply:
We are filtering out these 'generic' automated  genomes, and the gene names containing `made in|galleries`. I used this from the script at the bottom, and I believe @anandaroop added that. In my opinion this is a cleanup of *not important* genes (categories).

Here is an example of the categories:

```
    "categories": [
      "Sculpture",
      "Off the Wall",
      "Brass",
      "Wall Sculpture and Installation",
      "Use of Common Materials",
      "Smooth Surface",
      "Pastel Colors"
    ]
```